.github/actions/vtk-webassembly-install-dep/action.yml

```diff
@@ -66,31 +66,14 @@ runs:
         -DVTK_ENABLE_LOGGING=OFF
         -DVTK_ENABLE_REMOTE_MODULES=OFF
         -DVTK_ENABLE_WRAPPING=OFF
-        -DVTK_GROUP_ENABLE_Rendering=DEFAULT
-        -DVTK_GROUP_ENABLE_StandAlone=DEFAULT
         -DVTK_LEGACY_REMOVE=ON
-        -DVTK_MODULE_ENABLE_VTK_CommonSystem=YES
-        -DVTK_MODULE_ENABLE_VTK_FiltersGeneral=YES
-        -DVTK_MODULE_ENABLE_VTK_FiltersGeometry=YES
-        -DVTK_MODULE_ENABLE_VTK_IOCityGML=YES
-        -DVTK_MODULE_ENABLE_VTK_IOGeometry=YES
-        -DVTK_MODULE_ENABLE_VTK_IOImage=YES
-        -DVTK_MODULE_ENABLE_VTK_IOImport=YES
-        -DVTK_MODULE_ENABLE_VTK_IOPLY=YES
-        -DVTK_MODULE_ENABLE_VTK_IOParallel=YES
-        -DVTK_MODULE_ENABLE_VTK_IOXML=YES
-        -DVTK_MODULE_ENABLE_VTK_ImagingCore=YES
-        -DVTK_MODULE_ENABLE_VTK_ImagingHybrid=YES
-        -DVTK_MODULE_ENABLE_VTK_InteractionStyle=YES
+        -DVTK_MODULE_ENABLE_VTK_RenderingContext2D=YES
         -DVTK_MODULE_ENABLE_VTK_InteractionWidgets=YES
-        -DVTK_MODULE_ENABLE_VTK_RenderingAnnotation=YES
-        -DVTK_MODULE_ENABLE_VTK_RenderingCore=YES
-        -DVTK_MODULE_ENABLE_VTK_RenderingLabel=YES
-        -DVTK_MODULE_ENABLE_VTK_RenderingOpenGL2=YES
-        -DVTK_MODULE_ENABLE_VTK_RenderingVolumeOpenGL2=YES
+        -DVTK_MODULE_ENABLE_VTK_RenderingLICOpenGL2=NO
+        -DVTK_MODULE_ENABLE_VTK_hdf5=NO
+        -DVTK_MODULE_ENABLE_VTK_libproj=NO
         -DVTK_MODULE_ENABLE_VTK_TestingCore=YES
-        -DVTK_USE_SDL2=ON
-        -DVTK_SMP_IMPLEMENTATION_TYPE=STDThread
+        -DVTK_SMP_IMPLEMENTATION_TYPE=Sequential
```

Comment:
why changing all these params ?

Reply:
`Sequential` is required because `std::thread` usage is causing an exception.  
Regarding all other modules, VTK is not building with the default CMake options, so I just enabled all the required modules manually, which make the compilation faster.